app/controllers/application_controller.rb

```diff
@@ -1,2 +1,20 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+
+  before_action :configure_permitted_parameters, if: :devise_controller?
+
+  protected
+
+  def configure_permitted_parameters
+    devise_parameter_sanitizer.permit(:account_update) do |u|
+      u.permit(
+        :email,
+        :password,
+        :password_confirmation,
+        :current_password,
+        billing_attributes: %i[first_name last_name address city zip country phone],
```

Comment:
Devise have to handle only logic for authentication, not for updating user attributes and it's relations.